src/Components/Artwork/index.tsx

```diff
@@ -1,15 +1,26 @@
 import * as React from "react"
-import styled, { css } from "styled-components"
+import styled, { css, StyledFunction } from "styled-components"
 
 import theme from "../../Assets/Theme"
 import ArtworkMetadata, { ArtworkMetadataProps } from "./Metadata"
 import createContainer, { RelayProps } from "./Relay"
 
-const Container = styled.div`
+interface SizeDivProps extends React.HTMLProps<HTMLDivElement> {
+  size?: number
+}
+
+interface SizeImgProps extends React.HTMLProps<HTMLImageElement> {
+  size?: number
+}
+
+const SizeDiv: StyledFunction<SizeDivProps> = styled.div
+const SizeImg: StyledFunction<SizeImgProps> = styled.img
+
+const Container = SizeDiv`
```

Comment:
I wasn't really sure why `size` needed to be passed down to the styled divs/imgs below since it didn't _look_ like they were being used anywhere so I'm happy to remove them if you'd like!

Reply:
I guess it may have been used for dynamic styling previously? I don’t see why it’s currently needed either, so 🔪